ipywidgets/widgets/widget_float.py

```diff
@@ -58,6 +58,39 @@ def _validate_max(self, proposal):
             self.value = max
         return max
 
+class _BoundedLogFloat(_Float):
+    max = CFloat(10.0, help="Max value").tag(sync=True)
```

Comment:
Can we clarify that the min/max are *exponents*, while the value is an actual number? That was confusing to me.